crypto/src/hash/mod.rs

```diff
@@ -23,12 +23,16 @@ pub use rescue::{Rp62_248, Rp64_256};
 ///
 /// This trait defined hash procedures for the following inputs:
 /// * A sequence of bytes.
+/// * Collision resistance of the hash function.
 /// * Two digests - this is intended for use in Merkle tree constructions.
 /// * A digests and a u64 value - this intended for use in PRNG or PoW contexts.
 pub trait Hasher {
     /// Specifies a digest type returned by this hasher.
     type Digest: Digest;
 
+    /// Collision resistance of the hash function being used.
```

Comment:
nit: I would say: "Collision resistance of this hash function measured in bits."